-1

Tested at: c59c1741a69c6e9d3963c2baa9c5c8bab528b353

You can see the results of the tests here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-14153/12502/summary.html

I found follow errors while testing this PR

**Failed tests:  ClangBuild**

* **Clang**:

I found a compilation error while trying to compile with clang: 
I used this command: 
 scram b vclean && scram build -k -j 16 USER_CXXFLAGS='-fsyntax-only' COMPILER='llvm compile'


 <pre>>> Compiling  /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_1_X_2016-04-19-1100/src/FWCore/Concurrency/test/test_xercesInitialize.cpp 
>> Compiling  /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_1_X_2016-04-19-1100/src/FWCore/Concurrency/test/waitingtasklist_t.cppunit.cpp 
>> Compiling  /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_1_X_2016-04-19-1100/src/FWCore/Concurrency/test/serialtaskqueue_t.cppunit.cpp 
In file included from /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_1_X_2016-04-19-1100/src/FWCore/Concurrency/src/WaitingTaskList.cc:20:
In file included from /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_1_X_2016-04-19-1100/src/FWCore/Concurrency/interface/WaitingTaskList.h:77:
/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_1_X_2016-04-19-1100/src/FWCore/Concurrency/interface/WaitingTask.h:61:26: error: no member named 'make_unique' in namespace 'std'
        auto temp = std::make_unique<std::exception_ptr>(iPtr);
                    ~~~~~^
/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_1_X_2016-04-19-1100/src/FWCore/Concurrency/interface/WaitingTask.h:61:56: error: expected '(' for function-style cast or type construction
        auto temp = std::make_unique<std::exception_ptr>(iPtr);
                                     ~~~~~~~~~~~~~~~~~~^
</pre>

--- a/FWCore/Concurrency/interface/WaitingTask.h
+++ b/FWCore/Concurrency/interface/WaitingTask.h
@@ -21,6 +21,7 @@
 // system include files
 #include <atomic>
 #include <exception>
+#include <memory>
 #include "tbb/task.h"
 
 // user include files

